public/css/main.css

```diff
@@ -1,208 +1,596 @@
-  display: block;
-}
-body {
-  line-height: 1;
-}
-ol, ul {
-  list-style: none;
-}
-blockquote, q {
-  quotes: none;
-}
-blockquote:before, blockquote:after,
-q:before, q:after {
-  content: '';
-  content: none;
-}
-table {
-  border-collapse: collapse;
-  border-spacing: 0;
-}
-
-/* ==================== MAIN CSS ==================== */
-body {
-  box-sizing: border-box;
-  font-family: 'Roboto', sans-serif;
-}
-
-/* ===== RESPONSIVE IMAGE ===== */
-img {
-  width: 100%;
-  height: auto;
-  display: block;
-  border: 0 none;
-}
-
-/* ===== BASE ===== */
-a,
-a:hover,
-a:active,
-a:visited {
-  text-decoration: none;
-  color: #fff;
-}
-h1 {
-  font-weight: bold;
-  font-size: 40px;
-}
-
-/* ==================== HEADER ==================== */
-header {
-  padding: 20px;
-  background-color: #555;
-  color: #fff;
-  text-align: center;
-}
-/* ===== ANIMATION FOR SEARCH ===== */
-@keyframes stretching {
-  0%{width: 0;}
-  100%{width: 80%;}
-}
-/* ===== SEARCH ===== */
-.search {
-  z-index: 100;
-}
-.search input,
-.fa-search {
-  height: 40px;
-  line-height: 40px;
-  color: #555;
-  margin-bottom: 10px;
-}
-input[type="search"] {
-  width: 0;
-  float: left;
-}
-input[type="search"]:focus {
-  width: 80%;
-  animation-name: stretching;
-  animation-duration: 1s;
-}
-.fa-search {
-  padding: 5px;
-  width: 20%;
-  float: left;
-  color: #fff;
-}
-label {
-  float: left;
-}
-
-/* =====  NAV  ===== */
-.nav ul {
-  font-size: 0;
-}
-
-.nav li {
-  height: 40px;
-  line-height: 40px;
-  font-size: large;
-  display: inline-block;
-  margin-top: 10px;
-  margin-left: 4px;
-}
-
-.nav a {
-  padding: 0 5px;
-  box-sizing: border-box;
-  text-decoration: none;
-  color: #fff;
-  display: block;
-  transition: .3s background-color;
-}
-
-.profile {
-    background-color: green;
-    border-radius: 5%;
-}
-
-.logout {
-    background-color: red;
-    border-radius: 5%;
-
-}
-.nav a:hover {
-  background-color: rgba(255,255,255,0.2);
-  color: #fff;
-}
-.nav a.active {
-  background-color: rgba(255,255,255,0.15);
-  color: #fff;
-  cursor: default;
-  box-sizing: border-box;
-}
-
-/* ==================== MOBILE ==================== */
-@media screen and (max-width: 768px) {
-  .nav li {
-    font-size: small;
-  }
-  .search input,
-  .fa-search {
-    height: 25px;
-    line-height: 25px;
-  }
-  .filter div {
-    display: inline-block;
-  }
-
-}
-
-/* ==================== FILTER ==================== */
-.filter {
-  margin: 50px 0;
-
-}
-
-.btn{
-  margin: 1px;
-}
-
-
-/* ===== GALLERY ===== */
-.item {
-  box-sizing: border-box;
-  padding: 0;
-  margin: 20px 0;
-  min-height: 100px;
-}
-.btn-more {
-  float: right;
-}
-/* ==================== FOOTER ==================== */
-footer {
-  padding: 20px 0;
-  width: 100%;
-  height: 70px;
-  background-color: #555;
-  color: #fff;
-}
-.contact-image{
-  float: right;
-  margin-left: 10px;
-  width: 2em;
-  height: 2em;
-}
+<<<<<<< bb6b19124827def31ec98a00db03597a71b8733f
```

Comment:
technical resolving code should be deleted:
`<<<<<<< bb6b19124827def31ec98a00db03597a71b8733f`